x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.test.tsx

```diff
@@ -295,4 +295,21 @@ describe('ConnectorsDropdown', () => {
     );
     expect(tooltips[0]).toBeInTheDocument();
   });
+
+  test('it should hide the "Add New Connector" button when the user lacks the capability to add a new connector', () => {
```

Comment:
Thanks for the changes! The test works for multiple reasons but it does not actually test the feature. First, you need to pass `appendAddConnectorButton={true}` to `appendAddConnectorButton`. Then you need to open the selection by doing `userEvent.click(screen.getByTestId('dropdown-connectors'));` and check if the button exists by doing `expect(screen.queryByTestId('dropdown-connector-add-connector')).not.toBeInTheDocument();`. This will make the test fail. You need to mock `useApplicationCapabilities` to return `{ actions: { crud: false } }`.